src/test/incremental/hashes/if_expressions.rs

```diff
@@ -191,7 +191,7 @@ pub fn add_else_branch_if_let(x: Option<u32>) -> u32 {
 }
 
 #[cfg(not(cfail1))]
-#[rustc_clean(cfg="cfail2", except="HirBody,typeck_tables_of")]
+#[rustc_clean(cfg="cfail2", except="HirBody")]
```

Comment:
nope, can't be the `_` pattern, because that exists here, too. What changed for `if let`?